src/client/components/ConsentsSubHeader.tsx

```diff
@@ -11,8 +11,6 @@ import {
 } from '@guardian/source-foundations';
 import { AutoRow, gridItemColumnConsents, gridRow } from '@/client/styles/Grid';
 import { CONSENTS_PAGES_ARR } from '@/client/models/ConsentsPages';
-import { CONSENTS_MAIN_COLOR } from '@/client/layouts/shared/Consents';
-import { SvgCheckmark } from '@guardian/source-react-components';
```

Comment:
This seems to have been removed in error? Results in `make dev` failing locally if I rebase off this branch, due to `<SvgCheckmark />` being used on line 259.